### PR TITLE
DOCSP-26355: startup mode

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-kafka-connector/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-26355
**Staging**
Main page: https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-26355-start-specified-time/source-connector/configuration-properties/startup/
Old page (added admonition): https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-26355-start-specified-time/source-connector/configuration-properties/copy-existing/
What's New: https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-26355-start-specified-time/whats-new/#std-label-kafka-connector-whats-new-1.9
Usage example: https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-26355-start-specified-time/source-connector/usage-examples/copy-existing-data/#std-label-source-usage-example-copy-existing-data
All properties subsection: https://docs-mongodbcom-staging.corp.mongodb.com/kafka-connector/docsworker-xlarge/DOCSP-26355-start-specified-time/source-connector/configuration-properties/all-properties/#startup-mode

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

**Review by stIncMale — CHANGES_REQUESTED** [review]

File: source/source-connector/configuration-properties.txt [hunk 1]
```diff
@@ -68,7 +68,7 @@ for more information on these settings.
    Kafka Topic </source-connector/configuration-properties/kafka-topic>
    Change Stream </source-connector/configuration-properties/change-stream>
    Output Format </source-connector/configuration-properties/output-format>
-   Copy Existing </source-connector/configuration-properties/copy-existing>
+   Startup Mode </source-connector/configuration-properties/copy-existing>
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/source-connector/configuration-properties.txt:71]: Thanks for clarifying!

File: source/source-connector/configuration-properties/copy-existing.txt [hunk 2]
```diff
@@ -73,7 +98,7 @@ Settings
 
           .. code-block:: none
 
-             copy.existing.namespace.regex=stats\.page.*
+             startup.mode.copy.existing.namespace.regex=stats\\.page.*
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/source-connector/configuration-properties/copy-existing.txt:101]: I added the second `\` to match what I saw on [this line](https://github.com/mongodb/mongo-kafka/blob/40ae109680c3ec1af5c03850b76154da91d87d02/src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java#L358) in the source.

**Review by rozza — CHANGES_REQUESTED** [review]
Looking good - added a couple of suggestions for improvements

File: source/whats-new.txt [hunk 3]
```diff
@@ -98,7 +98,7 @@ Source Connector
 - Added support for the
   :manual:`allow disk use </reference/command/aggregate/#std-label-aggregate-cmd-allowDiskUse>`
   field of the {+query-api+} in the copy existing aggregation with the
-  ``copy.existing.allow.disk.use`` :ref:`configuration property <source-configuration-copy-existing>`
+  ``copy.existing.allow.disk.use`` :ref:`configuration property <source-configuration-startup>`
```
[the remark this answers is not in the document]
Comment (rustagir) [reply source/whats-new.txt:101]: adding a 1.9 what's new section

**Review by rozza — APPROVED** [review]
LGTM

File: source/source-connector/configuration-properties/startup.txt [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+           (for example, ``{"$timestamp": {"t": 30, "i": 0}}``)
+
+   * - | **startup.mode.copy.existing.namespace.regex**
+     - | **Type:** string
+       |
+       | **Description:**
+       | Regular expression the connector uses to match namespaces from
+         which to copy data. A namespace describes the MongoDB database name
+         and collection separated by a period, e.g.
+         ``databaseName.collectionName``.
```
Comment (mongoKart) [inline source/source-connector/configuration-properties/startup.txt:97]: Issue: Style guide recommends avoiding Latin abbreviations

```suggestion
         and collection separated by a period (for example,
         ``databaseName.collectionName``).
```

File: source/source-connector/configuration-properties/startup.txt [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+       .. example::
+
+          In the following example, the regular expression setting matches
```
Comment (mongoKart) [inline source/source-connector/configuration-properties/startup.txt:101]: Issue: Compound modifier

Suggestion:

```suggestion
          In the following example, the regular-expression setting matches
```

File: source/source-connector/configuration-properties/startup.txt [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+       | **Default**: ``16000``
+       | **Accepted Values**: An integer
+
+   * - | **startup.mode.copy.existing.allow.disk.use**
+     - | **Type:** boolean
+       |
+       | **Description:**
+       | When set to ``true``, 
+         the connector sets the copy existing aggregation 
+         to use temporary disk storage.
```
Comment (mongoKart) [inline source/source-connector/configuration-properties/startup.txt:160]: Issue: Something about this phrasing sounds a little off.

Suggestion:

```suggestion
         the connector uses temporary disk storage for the copy existing aggregation 
```

Or:

```suggestion
         the connector instructs the copy existing aggregation 
         to use temporary disk storage. 
```

...or something similar.

**Review by mongoKart — CHANGES_REQUESTED** [review]
Awesome. A few really small changes.

**Review by mongoKart — APPROVED** [review]
LGTM!

**Pull request merged**